cluster/gce/gci/configure-helper.sh

```diff
@@ -92,6 +92,7 @@ function main() {
     create-master-etcd-auth
     create-master-etcd-apiserver-auth
     override-pv-recycler
+    create-default-network-template-volume-mount
```

Comment:
not sure what volume mount is for.

Reply:
for create parameters used by  volume and volumeMount in cluster/gce/manifests/kube-controller-manager.manifest.

{"name": "defaulttemplate","mountPath": "/home/kubernetes/kube-manifests/kubernetes/default-network.tmpl", "
readOnly": false},

{"name": "defaulttemplate","hostPath": {"path": "/home/kubernetes/kube-manifests/kubernetes/default-network.tmpl",
 "type": "File"}},